src/components/navigation/Sidebar.tsx

```diff
@@ -1,75 +1,97 @@
-import { Text, View } from "react-native"
+import { Linking, Text, View } from "react-native"
+import { toast } from "sonner-native"
+import DiscordLogo from "../svgs/DiscordLogo"
+import { LogoSvg } from "../svgs/LogoSvg"
 import IconButton from "../ui/IconButton"
-import { Linking } from "react-native"
+import LargeButton from "../ui/LargeButton"
 
 const Sidebar = (props: DrawerContentComponentProps) => {
+  const colors = useColors()
   const router = useRouter()
   const navigation = useNavigation()
   const closeDrawer = () => {
     navigation.dispatch(DrawerActions.closeDrawer())
   }
 
   return (
-    <View className=" pt-16 px-4 pb-6  flex-1 flex-col flex justify-between border-0  bg-background">
-      <View className="flex flex-row justify-between items-center w-full">
+    <View className=" pt-16 px-4 pb-6 flex-1 flex-col flex justify-between border-0 bg-background">
+      <View className="flex flex-row justify-between items-center w-full py-8">
         <View className="flex flex-row gap-4 justify-between items-center w-full">
-          <View className="flex flex-row items-center gap-2">
-            {/* TODO: add logo */}
-            <View className="w-16 h-16 bg-white rounded-xl" />
-            <Text className="text-3xl text-foreground">Elektronik</Text>
+          <View className="flex flex-row items-center gap-1">
+            <LogoSvg width="70px" height="70px" />
+            <Text className="text-3xl text-foreground font-pmedium">
+              Elektronik
+            </Text>
           </View>
 
           <IconButton
             LucideIcon={X}
-            iconColor="#B6B6D9"
+            iconColor={colors.foreground}
             onPress={closeDrawer}
           />
         </View>
       </View>
 
-      <DrawerContentScrollView {...props}>
-        {/* change  to button component */}
-        <DrawerItemList {...props} />
-        <DrawerItem
-          label={"Radio Elektron"}
+      <DrawerContentScrollView
+        contentContainerClassName="flex flex-col gap-4"
+        {...props}
+      >
+        <LargeButton
+          LucideIcon={Disc3}
+          iconColor={colors.foreground}
+          text="Radio Elektron"
           onPress={() => router.push("/(drawer)/(tabs)/radio")}
         />
+        <LargeButton
+          LucideIcon={Clover}
+          iconColor={colors.foreground}
+          text="Szczęsliwy Numerek"
+          onPress={() => toast("Dostępne wkrótce!")}
+        />
       </DrawerContentScrollView>
-      <View className=" p-4 flex flex-col ">
-        <View className="border-b border-zinc-600 flex w-full justify-around flex-row pb-4 ">
+      <View className="flex flex-col gap-4">
+        <View className="flex w-full justify-center gap-4 flex-row">
           <IconButton
             LucideIcon={Facebook}
-            iconColor="#7293FF"
+            iconColor={colors.primary}
             onPress={() =>
               Linking.openURL("https://www.facebook.com/zgelektronik")
```

Comment:
tych socialmediow nie ma już gdzieś w strapi?

Reply:
jak sie okazuje to nie i na stronie tez sa na twardo 
![image](https://github.com/user-attachments/assets/e47bfd90-de86-4522-bafa-2c89c14a967c)